Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -210,7 +210,10 @@ private let routers: [Router<Route>] = [
     <¢> get %> lit("logout") <% end,
 
   .pricing
-    <¢> get %> lit("pricing") %> queryParam("plan", opt(.string)) <%> queryParam("quantity", opt(.int)) <% end,
+    <¢> get %> lit("pricing")
+    %> (queryParam("plan", opt(.string)) <%> queryParam("quantity", opt(.int)))
+      .map(PartialIso.pricing >>> Optional.iso.some)
```

Comment:
here `PartialIso.pricing` is an iso `(String?, Int?) -> Pricing`, and by mapping with `PartialIso.pricing >>> Optional.iso.some` we guarantee that this succeeds in parsing even if parsing the pricing value fails.

`Optional.iso.some` is a really bad name, it should be moved to an extension on `PartialIso where B == A?`, but we can clean up later.